apps/backoffice-v2/src/domains/business-reports/components/Transactions/Transactions.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { FunctionComponent } from 'react';
+import { ArrowRight } from 'lucide-react';
+
+import { PremiumSvg } from '@/common/components/atoms/icons';
+import { Link } from 'react-router-dom';
+
+export const Transactions: FunctionComponent = () => (
+  <div className={`flex flex-row`}>
+    <img
+      alt={`Transaction Analysis`}
+      src={'/images/TransactionAnalysis.png'}
+      className={`h-[567px] max-h-[567px] w-[1236px] max-w-[1236px]`}
```

Comment:
Either use height/width or max width/height. Pretty certain only the width/height are in effect.